pypeit/specobj.py

```diff
@@ -373,7 +375,11 @@ def apply_flux_calib(self, wave_sens, sensfunc, exptime, telluric=None, extinct_
             try:
                 sensfunc_obs[wave_mask] = interpolate.interp1d(wave_sens, sensfunc, bounds_error=True)(wave[wave_mask])
             except ValueError:
```

Comment:
In the sensfunc construction I added parameters extrap_red and extrap_blue to control these extrapolations and would prefer that the users think about what they are doing, but I think I agree that having a parameter to override that defualt and avoid a crash is a good thing, since it will crash initially. Maybe you can just add a comment here in the warning like: 

"Your data extends beyond the bounds of your sensfunc. You should be adjusting the par['sensfunc']['extrap_blu'] and/or par['sensfunc']['extrap_red'] to extrapolate further and recreate your sensfunc. But we are  extrapolating per your direction. Good luck!""

Reply:
warning  message added.